Code/GraphMol/RascalMCES/RascalButinaCluster.cpp

```diff
@@ -0,0 +1,126 @@
+//
+// Copyright (C) David Cosgrove 2023
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+// This file contains an implementation of Butina clustering
+// (Butina JCICS 39 747-750 (1999)) using the RascalMCES
+// Johnson similarity metric.  It is largely a transliteration
+// of $RDBASE/rdkit/ML/Cluster/Butina.py.
+
+#include <algorithm>
+#include <iterator>
+#include <vector>
+#include <set>
+
+#include <GraphMol/ROMol.h>
+#include <GraphMol/RascalMCES/RascalClusterOptions.h>
+#include <GraphMol/RascalMCES/RascalDetails.h>
+
```

Comment:
I suspect that you can resolve the win32_dll CI failures by including `RascalMCES.h` in this file.

The general rule is that you need to include the file that has the EXPORT declaration in the file that has the definition